security/rbac/scripts/submit_datagen_pageviews_config_avro.sh

```diff
@@ -0,0 +1,34 @@
+#!/bin/bash
+
+CONNECT_HOST=localhost
+
+if [[ $1 ]];then
+    CONNECT_HOST=$1
+fi
+
+HEADER="Content-Type: application/json"
+DATA=$( cat << EOF
+{
+  "name": "datagen-pageviews",
+  "config": {
+    "connector.class": "io.confluent.kafka.connect.datagen.DatagenConnector",
+    "kafka.topic": "pageviews",
+    "quickstart": "pageviews",
+    "key.converter": "org.apache.kafka.connect.storage.StringConverter",
+    "value.converter": "io.confluent.connect.avro.AvroConverter",
+    "value.converter.schema.registry.url": "http://localhost:8081",
+    "value.converter.basic.auth.credentials.source": "USER_INFO",
+    "value.converter.basic.auth.user.info": "client:client1",
+    "max.interval": 100,
+    "producer.override.sasl.mechanism": "OAUTHBEARER",
```

Comment:
Following up on comments left on the `connect-avro-distributed.properties.delta` file--we should remove the `producer.override.sasl.mechanism` and `producer.override.sasl.login.callback.handler` configs from this config and they should instead be set in the worker file.

Reply:
Agree